drivers/hmis_external_apis/app/models/hmis_external_apis/external_forms/form_submission.rb

```diff
@@ -38,24 +38,28 @@ def self.from_raw_data(raw_data, object_key:, last_modified:, form_definition:)
       submission
     end
 
+    def cde_helper(data_source_id:, system_user_id:)
+      @cde_helper ||= HmisExternalApis::TcHmis::Importers::Loaders::CustomDataElementHelper.new(
```

Comment:
Does this look okay? I think the effect of this is to just use the `data_source_id` and `system_user_id` from the first CDED in the iteration. Hypothetically it could be different for every CDED but in practice it will not be different, right?

Reply:
So we shouldn't use that `CustomDataElementHelper` class here. I mentioned it only as an example but that was confusing, sorry about that! The reason it shouldn't be used is the loader code is only temporary; it will be removed after the current migration is complete.

I recommend that you extract the relevant code out of that helper. I think the only piece that's needed here is the behavior in the cde_value_fields() method that sets all the value_[type] attributes. That could probably be rolled into this class